test/acceptance/behave/README.md

```diff
@@ -187,28 +187,28 @@ The log is stored in `logs` folder (if this folder does not exist it is created)
-|                                                                                                                                           |
-|**entities folder**                                                                                                                        |
-| - create_entities                         |     559      | POST   | /v2/entities/                                        | Yes            |      
-| - list_all_entities                       |     200      | GET    | /v2/entities/                                        | No             | 
-|                                                                                                                                           |
-| - update_append_attribute_by_id           |     766      | POST   | /v2/entities/`<entity_id>`                           | Yes            |   
-| - list_an_entity_by_id                    |     209      | GET    | /v2/entities/`<entity_id>`                           | No             | 
-| - replace_attributes_by_id                |     511      | PUT    | /v2/entities/`<entity_id>`                           | Yes            |   
-| - update_attribute_by_id                  |     654      | PATCH  | /v2/entities/`<entity_id>`                           | Yes            | 
-| - delete_entity_by_id                     |  (pending)   | DELETE | /v2/entities/`<entity_id>`                           | No             |
-|                                                                                                                                           |
-| - list_an_attribute_by_id                 |     198      | GET    | /v2/entities/`<entity_id>`/attrs/`<attr_name>`       | No             |   
-| - update_only_one_attribute_by_id         |  (pending)   | PUT    | /v2/entities/`<entity_id>`/attrs/`<attr_name>`       | Yes            | 
-| - delete_attribute_by_id                  |  (pending)   | DELETE | /v2/entities/`<entity_id>`/attrs/`<attr_name>`       | No             |
-|                                                                                                                                           |
-| - update_only_one_attribute_value_by_id   |  (pending)   | PUT    | /v2/entities/`<entity_id>`/attrs/`<attr_name>`/value | Yes            |   
-|                                                                                                                                           |
-|**alarms folder**                          |  (pending)   |                                                                                |
+|           FEATURE                           |  TEST CASES  | METHOD  |            URL                                      |  PAYLOAD       |  
+|:--------------------------------------------|:------------:|--------:|:----------------------------------------------------|:--------------:|      
```

Comment:
It is a table, the requests are grouping by url and the empty rows are used as separator. The method is align to right at the moment. See in view mode.  https://github.com/telefonicaid/fiware-orion/blob/feature/delete_by_id_acceptance_tests/test/acceptance/behave/README.md